client/src/templates/Challenges/redux/execute-challenge-saga.js

```diff
@@ -78,19 +77,16 @@ export function* executeCancellableChallengeSaga(payload) {
     }
   }
 
-  if (previewTask) {
-    yield cancel(previewTask);
-  }
   // executeChallenge with payload containing {showCompletionModal}
   const task = yield fork(executeChallengeSaga, payload);
-  previewTask = yield fork(previewChallengeSaga, { flushLogs: false });
+  yield fork(previewChallengeSaga, { flushLogs: false });
```

Comment:
```suggestion
```
Even though `previewTask` is not used, forking the `previewChallengeSaga` still causes it to be run.

Reply:
Updated the PR.